src/cljs/main/broadfcui/page/workspace/data/import_data.cljs

```diff
@@ -32,7 +32,8 @@
                                (.readAsText reader (.slice file 0 preview-limit)))))}]
       [:div {:style {:marginBottom "0.8em"}} "For more information, see our "
        [:a {:href (config/user-guide-url) :target "_blank"} "user guide"] "."]
-      [comps/Button {:text "Choose file..." :onClick #(-> (@refs "entities") .click)}]
+      [comps/Button {:text (if (:upload-result @state) "Choose another file..." "Choose file...")
+                     :onClick #(-> (@refs "entities") .click)}]
```

Comment:
After uploading a file that gave me an error, choosing another one didn't work. Probably not your fault but could you file the bug please?